source/content/guides/wordpress-configurations/07-wordpress-font-library.md

```diff
@@ -0,0 +1,87 @@
+
+</Alert>
+
+## Considerations
+
+The distinction between the two approaches for handling fonts depends what works best for your workflow and whether you consider your fonts to be handled more like media files or more like themes and plugins.
+
+**If you consider fonts to be more like media files**, use the default Pantheon behavior added in our `mu-plugin`. This makes it easier to _install fonts onto production sites_ but requires you to pull your database and files to the lower environments from Live. See [our documentation about database workflows](https://docs.pantheon.io/guides/mariadb-mysql/database-workflow-tools) for more information.
+
+**If you consider fonts to be more like themes and plugins**, use the WordPress default behavior and the `pantheon_modify_fonts_dir` filter. This ensures that the fonts are installed across environments, but does not necessarily mean that fonts that are _physically installed_ on your lower environments (e.g. they exist in the `wp-content/fonts` directory) are recognized by WordPress. (They may still need to be added in the WordPress admin.)
+
+## Troubleshooting
+
+### "No font faces were installed" error message
+
+![No font faces were installed](../../../images/wordpress-configurations/07-no-font-faces-installed.png)
+
+There are two reasons you might receive an error message like the above that says that no fonts were installed and asks if the parent directory is writable by the server, both assume you have _disabled_ our modification to the directory where fonts are installed and are using the WordPress core handling (uploading to `wp-content/fonts/`).
+
+1. The site environment is Dev and is in Git mode. If the Dev environment is set to Git mode, the filesystem is not writeable and you will not be able to install fonts.
+2. The site environment is Test or Live and you've set the `'pantheon_modify_fonts_dir'` filter value to `false`. In this case, you will not be able to install fonts on any environment other than Dev and you will need to commit them to your codebase.
+
+### I've cloned my database from Live and my font is installed but not "active"
+
+![Font is installed but not active](../../../images/wordpress-configurations/07-font-installed-not-active.png)
+
+If you've cloned your database and files from your live environment to Dev or Test and the font appears in your Font Library but is not "active" (it displays a message like "0/1 variants active"), you can click into the font, select the variant you want, and click update. 
+
+![Update Font](../../../images/wordpress-configurations/07-font-update.png)
+
+### I've uploaded fonts to my `/fonts` directory, but they aren't showing up in the Font Library
+
+Fonts need to be installed via the Font Library in the WordPress admin. This is because, in the backend, they are stored as a post type. Therefore, simply having a font in your `/fonts` directory does not necessarily mean it will be recognized by WordPress.
```

Comment:
Isn't this the state if we're tracking fonts in version control and pushing up "as code" then? What is the next step then?

Reply:
Good point. I guess you'd need to install it via the admin (or in code) again? I'm still waiting for clarification in WP core Slack because the other option would be cloning the DB from a lower environment. I tried exporting a WXR and importing it into a local copy of the same site and I got a Gutenberg error when I tried to open the font library, so that's a no-go. When you upload a font into the Font Library where the files already exist, you get an error (that a font with that name already exists) but it _does_ show up in the font library. 

There's an open ticket dealing with fonts still being listed after the _files_ have been deleted: https://github.com/WordPress/gutenberg/issues/58375

There's also an open ticket for exporting fonts in a WP Export: https://github.com/WordPress/gutenberg/issues/54226 -- this one might be the missing next step, particularly if you could export _just_ fonts.

There isn't (yet) a ticket dealing with font _files_ existing but I thought I remembered seeing a discussion about it. I will open an issue since my Slack thread hasn't gotten any attention.

Reply:
https://github.com/WordPress/gutenberg/issues/59102